lib/pages/chat_draft/draft_chat.dart

```diff
@@ -260,6 +259,40 @@ class DraftChatController extends State<DraftChat>
     );
   }
 
+  @override
+  void sendFileOnWebAction(
+    BuildContext context, {
+    Room? room,
```

Comment:
remove this also

Reply:
To override it has to have the same signature